js/pluralizer.js

```diff
@@ -0,0 +1,22 @@
+
+let thing = "cat"
+
+let count = Math.floor((Math.random() * 5) + 1)
+
+if (count > 1) {
+    let thing = "cats"
+}
+
+if (count == 1) {
+    console.log("1", thing)
+    } else if (count == 2) {
+        console.log("2", thing)
+    } else if (count == 3) {
+        console.log("3", thing) 
+    } else if (count == 4) {
+        console.log("4", thing) 
+    } else if (count == 5) {
+        console.log("5", thing) 
+}
```

Comment:
you need some string interpolation my friend https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals
start with back tick ` text ${variable } more text ` close with back tick  | all 1 string